### PR TITLE
Move Scout package ecosystem section

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Proposed changes

Consolidates package ecosystem into newer details page.

### CODE REVIEW

**Review by aevesdocker — DISMISSED** [review]

**Review by dvdksn — APPROVED** [review]

File: scout/advisory-db-sources.md [hunk 1]
```diff
@@ -46,6 +46,21 @@ trackers, such as:
 - [Ubuntu CVE Tracker](https://people.canonical.com/~ubuntu-security/cve/)
 - [Wolfi Security Feed](https://packages.wolfi.dev/os/security.json)
 
+And the following package ecosystems:
```
[the remark this answers is not in the document]
Comment (jonnyrobots) [reply scout/advisory-db-sources.md:49]: Or perhaps it's simply a new section heading?
Eg. "Package ecosystems supported by the Docker Scout Advisory Database"

[the remark this answers is not in the document]
Comment (ChrisChinchilla) [reply scout/advisory-db-sources.md]: @jonnyrobots OK, I misread the earlier discussions. Done and for now moved to the bottom as it felt "disruptive" to have it in the existing sections, but can find a way to move it up if you think that's better.

File: scout/advisory-db-sources.md [hunk 2]
```diff
@@ -46,6 +46,8 @@ trackers, such as:
 - [Ubuntu CVE Tracker](https://people.canonical.com/~ubuntu-security/cve/)
 - [Wolfi Security Feed](https://packages.wolfi.dev/os/security.json)
 
+- `deb` (Debian Linux and derivatives)
```
Comment (dvdksn) [inline scout/advisory-db-sources.md:49]: I think we forgot this little fella :)

Comment (ChrisChinchilla) [reply scout/advisory-db-sources.md:49]: Debian deserves it's own place :)

**Review by dvdksn — DISMISSED** [review]
LGTM - please squash!

**Review by jonnyrobots — CHANGES_REQUESTED** [review]
Not strictly related to this PR, but in the same file (`advisory-db-sources.md`); can we replace the word scanning, please?

> Docker Scout is ideal for ~scanning~ images in Docker Desktop and Docker Hub
to
> Docker Scout is ideal for *analysing* images in Docker Desktop and Docker Hub

**Comment by ChrisChinchilla** [comment]
@jonnyrobots It's a good catch.

**Review by aevesdocker — APPROVED** [review]

**Pull request merged**